migrations/sqls/20201026081506-add-charge-version-licence-id-up.sql

```diff
@@ -0,0 +1,9 @@
+/* Add the licence id to charge versions */
+ALTER TABLE water.charge_versions
+  ADD COLUMN IF NOT EXISTS "licence_id" uuid;
```

Comment:
Should this have a `NOT NULL` constraint?

Reply:
If so, this will have knock on effects with the charge version workflow in the UI and the `postCreateFromWorkflow` endpoint in the `charge-versions` module

Reply:
I had to leave the not null out because I noticed there are some charge versions that don't have a licence ref. This means the licence id can not be populated. But more importantly the import feature will fail because it copies the charge version data from NALD and then it updates the licence id. 

The import SQL statement could be changed but this is already so complex I didn't want to add another join to it only to get the licence id. When we switch the charging import feature off we could add the constraint and at the same time remove the licence ref as this is not really needed if we will use the licence id instead.